python/telemetry/cirbe_70cm.py

```diff
@@ -42,8 +43,16 @@ def _encode(self, obj, context, path=None):
         return Container()
 
     def _decode(self, obj, context, path=None):
-        offset = datetime(2000, 1, 1, 12) - datetime(1970, 1, 1)
-        return (datetime.utcfromtimestamp(obj.time_stamp_seconds) + offset)
+        t = datetime.datetime.fromtimestamp(
+            obj.time_stamp_seconds,
+            tz=datetime.timezone.utc
+        )
+        t = t.replace(tzinfo=None)
+        offset = (
+            datetime.datetime(2000, 1, 1, 12)
+            - datetime.datetime(1970, 1, 1)
+        )
+        return (t + offset)
```

Comment:
I'm looking at what the original code did and it is a badly written (but correct) implementation of getting a `datetime` that is `obj.time_stamp_seconds` after the J2000 epoch (which is 2000-01-01T12:00:00Z). Therefore, the following implementation seems much more clear:
```python
j2000_epoch = datetime.datetime(2000, 1, 1, 12, tzinfo=datetime.datetime.utc)
return j2000_epoch + datetime.timedelta(seconds=obj.time_stamp_seconds)
```

I would probably return a timezone-aware datetime here.